osx/KA-Lite Monitor/KA-Lite Monitor.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,466 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 46;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		F103615C1A81B7DF00998E6B /* images in Resources */ = {isa = PBXBuildFile; fileRef = F103615B1A81B7DF00998E6B /* images */; };
+		F1720BED1A79566700096453 /* README.md in Sources */ = {isa = PBXBuildFile; fileRef = F1720BEC1A79566700096453 /* README.md */; };
+		F1720BF11A7A343A00096453 /* local_settings.default in Resources */ = {isa = PBXBuildFile; fileRef = F1720BF01A7A343A00096453 /* local_settings.default */; };
+		F1B308371ACBD08400008AF1 /* ka-lite in Resources */ = {isa = PBXBuildFile; fileRef = F1B308361ACBD08300008AF1 /* ka-lite */; };
+		F1B308391ACBD09500008AF1 /* pyrun-2.7 in Resources */ = {isa = PBXBuildFile; fileRef = F1B308381ACBD09400008AF1 /* pyrun-2.7 */; };
+		F1DDD4131A6E5DC10022F1C9 /* AppDelegate.m in Sources */ = {isa = PBXBuildFile; fileRef = F1DDD4121A6E5DC10022F1C9 /* AppDelegate.m */; };
+		F1DDD4151A6E5DC10022F1C9 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = F1DDD4141A6E5DC10022F1C9 /* main.m */; };
+		F1DDD4171A6E5DC10022F1C9 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = F1DDD4161A6E5DC10022F1C9 /* Images.xcassets */; };
+		F1DDD4261A6E5DC10022F1C9 /* KA_Lite_MonitorTests.m in Sources */ = {isa = PBXBuildFile; fileRef = F1DDD4251A6E5DC10022F1C9 /* KA_Lite_MonitorTests.m */; };
+		F1E5A30B1A8041FD003FA650 /* MainMenu.xib in Resources */ = {isa = PBXBuildFile; fileRef = F1E5A3091A8041FD003FA650 /* MainMenu.xib */; };
+/* End PBXBuildFile section */
+
+/* Begin PBXContainerItemProxy section */
+		F1DDD4201A6E5DC10022F1C9 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = F1DDD4041A6E5DC10022F1C9 /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = F1DDD40B1A6E5DC10022F1C9;
+			remoteInfo = "KA-Lite Monitor";
+		};
+/* End PBXContainerItemProxy section */
+
+/* Begin PBXFileReference section */
+		F103615B1A81B7DF00998E6B /* images */ = {isa = PBXFileReference; lastKnownFileType = folder; name = images; path = Resources/images; sourceTree = "<group>"; };
```

Comment:
These are automatically added by Xcode's IDE - they are there because I have added the `ka-lite` and `pyrun-2.7` folders as Resources for the project.